### DISCUSSION TITLE
Qodana for windows agents

### DISCUSSION THREAD

**Discussion by uejo** [JetBrains/Qodana#165] [Q&A]
Hello, 
is there an eta for Qodana plugin to be used on windows agents? Currently we get the
`teamcity.agent.jvm.os.name does not contain Windows ` message as incompatible agent requirements.
We know we can use the qodana.exe for analysis, but we would like the integrated version to use all the features the plugin can offer.
Thanks!

**Comment by brichbash** [comment]
Hello,

Unfortunately, we have no ETA for it yet. We're planning to start using qodana-cli in the plugin because it would allow using Windows agents. If you have a spare minute, please vote for the issue [on our tracker](https://youtrack.jetbrains.com/issue/QD-4136/Support-Qodana-runner-on-Windows-agents) to get notified when there are updates.

**Reply by Ssteve85uk** [reply]
Is this now available or is it coming soon?

**Reply by zsd4yr** [reply]
Looks like qodana clii works on windows but requires docker :( 
qodana scan
✗  Docker (or podman) is not installed on the system or can't be found in PATH, refer to https://www.docker.com/get-started for installing it
